### ISSUE TITLE
Change of behavior in parsing additional file arguments.

### ISSUE THREAD

**Issue by budiatmadjajaWill** [mikedh/trimesh#1821]
Thank you for the great library!

I notice that there's a change in behavior between release 3.15.8 to 3.16.0 in trimesh/exchange/ply.py. Especially in the `load_element_single(properties, data)` funciton. 

Currently when trying to load a ply mesh with face information: 
`3 0 1 2 0 0`

from what I understand, the first digit should be the number of face elements `3`, then `0 1 2` should be the vertex indices and the last two values, `0 0` should be the additional file arguments. However, in the newer version the additional file arguments starts from `2` instead of `0`. 

Changing the number of face elements to `4` solves the issue but that's not the right way to specify it. 

Is this behavior expected? 

Thank you in advance!

**Comment by mikedh** [comment]
Hey, have you tried using `trimesh.load('file.ply')`? The functions in `trimesh.exchange.ply` are pretty much internal and are probably not what you want.

**Comment by budiatmadjajaWill** [comment]
Hi, thank you for the reply! Sorry for explaining it quite poorly, I am using the `trimesh.load("file.ply")` function then I did some looking in `trimesh.echange.ply` since the behavior changed.

**Comment by mikedh** [comment]
Oh gotcha, yeah I think I will add an underscore to those function definitions so they don't get added to docs. I don't think anything changed in PLY recently, although something upstream might have.

Are you able to reduce it to a minimal example that fails? Something like this:
```
import numpy as np
import trimesh
mesh = trimesh.load('box.ply')
assert np.isclose(mesh.volume, 1.0)
```

**Comment by budiatmadjajaWill** [comment]
Cool! I will add a reproducible example in a couple of days. Talk to you soon.

**Comment by budiatmadjajaWill** [comment]
Hi @mikedh I've come back with the reproducible example. 
To simplify the problem I had, we'll just use a mesh file with 2 faces inside the zip files here:
[trimesh_metadata_example.zip](https://github.com/mikedh/trimesh/files/10556982/trimesh_metadata_example.zip)
There is an additional property added to the face data called face_type and the first triangle has its property set to 12 and the second triangle has its property set to 90

```
import trimesh
print(f"Trimesh version: {trimesh.__version__}")
test = trimesh.load('../../../trimesh_metadata_example.ply')
print(f"face_type:\n\t{test.metadata['_ply_raw']['face']['data']['face_type'].tolist()}")
```
With trimesh version 3.15.8: 
<img width="145" alt="image" src="https://user-images.githubusercontent.com/85486615/216044762-75775541-f892-4f67-8ff5-c6ab3b88312e.png">

With trimesh version 3.16.0:
<img width="145" alt="image" src="https://user-images.githubusercontent.com/85486615/216045526-e14fc63c-ed50-4eda-bbf2-ef9a588cf9d9.png">

If you open the ply file, you can see that [1] for the first triangle and [3] for the second triangle are the third vertex index for each triangles. 

Thank you!

**Comment by mikedh** [comment]
Thanks for the example! Looks like an off-by-one index error for custom data *after* a list, in ASCII PLY files. Should be fixed by #1823

**Comment by budiatmadjajaWill** [comment]
Great, thank you for the response! 
I think that should be enough for a fix. I'll keep this issue opened for it to be automatically closed with the Pull Request. 

Have a nice day!
